### PR TITLE
Pass desired version to SGX SDK action

### PR DESCRIPTION
**Pull request by nick-mobilecoin**

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/actions/2" target="_blank">Graphite</a>  👈
    * **PR #3** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/actions/3" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/actions/2?utm_source=stack-comment).

File: sgxsdk/action.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 name: Install SGX SDK
 description: Installs the Intel SGX SDK (and propagates environment variables)
+inputs: 
+  version:
+    description: "The version of the SDK to install"
```
Comment (nick-mobilecoin) [inline sgxsdk/action.yaml:5]: Usage can be seen on, https://github.com/mobilecoinfoundation/sgx/pull/213

File: sgxsdk/action.yaml [hunk 2]
```diff
@@ -14,9 +19,9 @@ runs:
       # fail on.  So we manually pass on the known environment variables.
       run:
         source /etc/lsb-release &&
-        wget "https://download.01.org/intel-sgx/sgx-linux/2.17.1/distro/ubuntu${DISTRIB_RELEASE}-server/sgx_linux_x64_sdk_2.17.101.1.bin" &&
-        chmod +x ./sgx_linux_x64_sdk_2.17.101.1.bin &&
-        ./sgx_linux_x64_sdk_2.17.101.1.bin --prefix=/opt/intel &&
+        wget "https://download.01.org/intel-sgx/sgx-linux/`echo ${{ inputs.version }} | cut -d '.' -f1-2`/distro/ubuntu${DISTRIB_RELEASE}-server/sgx_linux_x64_sdk_${{ inputs.version }}.bin" &&
```
Comment (nick-mobilecoin) [inline sgxsdk/action.yaml:22]: I'm up for better alternatives to the `echo ${{ inputs.version }} | cut -d '.' -f1-2`, my bash is weak and it was one of the first solutions I saw to get the `maj.min` out of the version string

**Review by jcape — APPROVED** [review]

**Review by awygle — APPROVED** [review]
This seems fine, I don't have a problem with that Bash command.

**Review by NotGyro — APPROVED** [review]
Looks good to me! This is a welcome change.

**Pull request merged**